crates/task-impls/src/consensus.rs

```diff
@@ -1075,6 +1096,8 @@ where
                     "We received a timeout event in the consensus task for view {}!",
                     *view
                 );
+                let consensus = self.consensus.read().await;
```

Comment:
Where do we reset this metric?  I think we should reset it once we have a view that succeeds.

Reply:
This is for resetting the consensus metric `number_of_timeouts`, and will not be used for any other place. The whole consensus thing is wrapped in an Arc,  therefore as long as we know which one to change it should be okay (like here we simply want to get `self.consensus`).